api/templates/api/base.html

```diff
@@ -0,0 +1,20 @@
+<!-- templates/front/base.html -->
```

Comment:
Why do you want to create templates view in api aplication? 
Do you think that this app should have own view? Why?

Reply:
deleted